client/keys/import.go

```diff
@@ -38,3 +38,19 @@ func ImportKeyCommand() *cobra.Command {
 		},
 	}
 }
+
+func ImportKeyHexCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "import-hex <name> <hex> <key-type>",
+		Short: "Import private keys into the local keybase",
+		Long:  "Import hex encoded private key into the local keybase.",
```

Comment:
Can we add that the supported key types can be gotten from `<appd> list-key-types`?